exts/cesium.omniverse/cesium/omniverse/extension.py

```diff
@@ -445,3 +450,19 @@ def _open_window():
     @staticmethod
     def _open_modal():
         CesiumFabricModal()
+
+    def _setup_ion_server_prims(self):
+        # TODO: Move a lot of this to usdUtils.py
```

Comment:
Note to self to address this TODO

Reply:
I think I'm just going to leave this TODO for now. I might address it in a future PR where there's more Python changes.